wcfsetup/install/files/lib/action/PaypalCallbackAction.class.php

```diff
@@ -55,8 +55,8 @@ public function execute() {
 			}
 			
 			// Check that receiver_email is your Primary PayPal email
-			if (strtolower($_POST['receiver_email']) != strtolower(PAYPAL_EMAIL_ADDRESS)) {
-				throw new SystemException("Mismatching receiver_email '" . $_POST['receiver_email'] . "', expected '".PAYPAL_EMAIL_ADDRESS."'.");
+			if (strtolower($_POST['receiver_email']) != strtolower(PAYPAL_EMAIL_ADDRESS) && strtolower($_POST['business']) != strtolower(PAYPAL_EMAIL_ADDRESS)) {
+				throw new SystemException("Mismatching receiver_email and business '" . $_POST['receiver_email'] . "', expected '".PAYPAL_EMAIL_ADDRESS."'.");
```

Comment:
I can't comment on whether the change is correct, because I don't have the necessary PayPal expertise, but in any case the `receiver_email` in the exception message should also be adjusted.

Reply:
This part existed before but was removed: https://github.com/WoltLab/WCF/commit/4b200b5e9a1534d34465d03afe509df98a18a496
While it's not a requirement it's still possible to use it.

I'm using a seperate email for the forum to seperate it with other payments which resulted in this.

Reply:
I adjusted the exception message to include the business email